### ISSUE TITLE
Bug: Creating new whiteboards fails with "Not a diagram file"

### ISSUE THREAD

**Issue by arnowelzel** [jgraph/drawio-nextcloud#59]
When I create a new diagram, the diagram will open with the following error message:

![image](https://github.com/jgraph/drawio-nextcloud/assets/6613614/bd24aacb-0ae2-4159-aac7-0f8f610b3ded)

When I confirm that with "OK", I can even draw into the diagram and also click "Save", but it seems not to get saved at all. The file exists but just stays empty.

Maybe this is a side effect of the fix in https://github.com/jgraph/drawio-nextcloud/commit/8de1b20fd09755836c3d508f74372436a7998678? Because when I add the following content to the new ".dwb"-file, it works without any problem:

```
<mxGraphModel><root><mxCell id="0"/><mxCell id="1" parent="0"/></root></mxGraphModel>
```

**Comment by arnowelzel** [comment]
Ok, it seems for whiteboards the file must not be empty, but for diagrams it must be empty. I'll modified the code in that way, that it will use the default XML content for new whiteboards but not for new diagrams. So far this works OK here. I'll prepare a PR for that.

**Comment by m-mohamedin** [comment]
Thanks for the fix, please use v2.1.4
